charts/minibroker/templates/deployment.yaml

```diff
@@ -71,3 +74,12 @@ spec:
           initialDelaySeconds: 10
           periodSeconds: 15
           failureThreshold: 2
+        volumeMounts:
+        - name: override-params
+          mountPath: {{ $configPath | quote}}
+          readOnly: true
+      volumes:
+      - name: override-params
+        configMap:
+          name: {{ printf "%s-override-chart-params" .Release.Name | quote }}
+          defaultMode: 0755
```

Comment:
I falsely assumed this setting is about the directory (where we'd need the `+x` flags) not the files. `0400` doesn't work though because we're running as the `minibroker` user now. I'll just remove this line to make it use the default of `0644` which works nicely.